packages/fluentui/react-northstar/src/components/Tree/Tree.tsx

```diff
@@ -253,47 +358,46 @@ class Tree extends AutoControlledComponent<WithAsProp<TreeProps>, TreeState> {
   };
 
   renderContent(accessibility: ReactAccessibilityBehavior): React.ReactElement[] {
-    const { items, renderItemTitle } = this.props;
+    const { items, renderItemTitle, selectable } = this.props;
 
     if (!items) return null;
 
-    const renderItems = (
-      items: ShorthandCollection<TreeItemProps>,
-      level = 1,
-      parent?: string,
-    ): React.ReactElement[] => {
-      return items.reduce((renderedItems: React.ReactElement[], item: ShorthandValue<TreeItemProps>, index: number) => {
-        const itemId = item['id'];
+    const renderItems = (items: TreeItemProps[], level = 1, parent?: string): React.ReactElement[] => {
+      return items.reduce((renderedItems: React.ReactElement[], item: TreeItemProps, index: number) => {
+        const { id } = item;
         const isSubtree = hasSubtree(item);
-        const isSubtreeExpanded = isSubtree && this.isActiveItem(itemId);
+        const isSubtreeExpanded = isSubtree && this.isActiveItem(id);
+        const isSelectedItem = this.isSelectedItem(id);
 
-        if (!this.itemsRef.has(itemId)) {
-          this.itemsRef.set(itemId, React.createRef<HTMLElement>());
+        if (!this.itemsRef.has(id)) {
+          this.itemsRef.set(id, React.createRef<HTMLElement>());
         }
 
         const renderedItem = TreeItem.create(item, {
           defaultProps: () => ({
             accessibility: accessibility.childBehaviors ? accessibility.childBehaviors.item : undefined,
             expanded: isSubtreeExpanded,
+            selected: isSelectedItem,
+            selectable,
             renderItemTitle,
-            key: item['id'],
+            key: id,
             parent,
             level,
             index: index + 1, // Used for aria-posinset and it's 1-based.
-            contentRef: this.itemsRef.get(itemId),
+            contentRef: this.itemsRef.get(id),
             treeSize: items.length,
           }),
         });
 
         return [
           ...renderedItems,
           renderedItem,
-          ...(isSubtreeExpanded ? renderItems(item['items'], level + 1, itemId) : ([] as any)),
+          ...(isSubtreeExpanded ? renderItems(item.items as TreeItemProps[], level + 1, id) : ([] as any)),
```

Comment:
This can be problematic as the shorthands can also be strings

Reply:
if it is string it is basically: {mappedProp: value}, whatever the mappedProp inside TreeItemProps is

Reply:
I had call with Marija :)
thanks for find it out.
Reported the bug regarding it
https://dev.azure.com/microsoftdesign/fluent-ui/_workitems/edit/7102